pkg/visor/rpc_client_serve.go

```diff
@@ -22,15 +24,32 @@ func isDone(ctx context.Context) bool {
 }
 
 // ServeRPCClient repetitively dials to a remote dmsg address and serves a RPC server to that address.
-func ServeRPCClient(ctx context.Context, log logrus.FieldLogger, dmsgC *dmsg.Client, rpcS *rpc.Server, rAddr dmsg.Addr, errCh chan<- error) {
+func ServeRPCClient(ctx context.Context, log logrus.FieldLogger, autoPeer bool, autoPeerCmd string, dmsgC *dmsg.Client, rpcS *rpc.Server, rAddr dmsg.Addr, errCh chan<- error) {
```

Comment:
Instead of passing both a autoPeer bool and cmd string, we could just pass the string and check if its empty? Also, if I read the code correctly, the autoPeerCmd is just the IP+Port, in which case the variable name is misleading.

Reply:
I changed what the variables were doing without changing their name

I've altered it to your suggestion
[a2a32523fe86bb3ff2f45890471c4070ca929566](https://github.com/skycoin/skywire/pull/1309/commits/a2a32523fe86bb3ff2f45890471c4070ca929566)
[920493eb40531523973654465284259aa1b6956f](https://github.com/skycoin/skywire/pull/1309/commits/920493eb40531523973654465284259aa1b6956f)